src/openrct2/windows/ride_list.c

```diff
@@ -411,7 +421,13 @@ static void window_ride_list_scrollmousedown(rct_window *w, int scrollIndex, int
 		return;
 
 	// Open ride window
-	window_ride_main_open(w->list_item_positions[index]);
```

Comment:
>Something I would like to encourage is stashing as much as possible in locals, particularly if it reduces the same expression multiple times. So in this case you can do:
uint8 rideIndex = w->list_item_positions[index];
>
>And use rideIndex below in the two blocks of code instead of w->list_item_positions[index]

I've done as you have said. I probably would have originally done this, but in all honesty I kinda just threw this PR together originally while discussing it in the Mozart vandalism server. Thanks for pointing it out. I put `uint8 rideIndex = w->list_item_positions[index];` right below the `//open ride window`, is that okay? or should i declare the `uint8 rideIndex` at the beginning and then assign it below the comment? I'm unsure if declaring as functions go along causes any difference. (I know this is premature optimization worries, but i've just always been curious)